tomcat/src/main/java/org/apache/coyote/http11/StringTokenizer.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.coyote.http11;
+
+public class StringTokenizer {
+
+    public static String[] token(final String str, final String delimiter) {
+        return str.split(delimiter);
+    }
+}
```

Comment:
```suggestion
public final class StringTokenizer {
    private StringTokenizer() {}

    public static String[] token(final String str, final String delimiter) {
        return str.split(delimiter);
    }
}
```

이건 취향 차이라서 꼭 반영할 필요는 없는데,
저는 유틸성 클래스의 경우에는 인스턴스 생성과 상속이 불가능하도록,
final 키워드와 private 생성자를 명시해 주는 것을 좋아하는 편입니다.

Reply:
2단계 진행하면서 리팩터링 반영해보겠습니다. 감사합니다 🙏